### PR TITLE
Feat ryan gosling biography added

### PR DESCRIPTION
**Pull request by abrilmunozc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <p>
+        <h2><strong>Name:</strong> <em>Ryan Gosling</em></h2></p>
+        <p><strong>Age:</strong> </body> 42 years old</p>
+        <p><strong>Date of birth: <mark> November 12, 1980 </mark></strong>  </p>
+        <p><strong>Nationality:</strong> Canadian</p>
+        <p><strong>Description:</strong> Ryan Thomas Gosling (born November 12, <ins>1980</ins>) is a Canadian actor. Prominent in both independent film and major studio features of varying genres, his films have accrued a worldwide box office gross of over 1.9 billion USD. He has received various accolades, including a Golden Globe Award, and nominations for two Academy Awards and a BAFTA Award.</p>
+        <p>Born and raised in Canada, he rose to prominence at age 13 as a child actor on Disney Channel's The Mickey Mouse Club <ins>(1993–1995),</ins> and went on to appear in other family entertainment programs, including Are You Afraid of the Dark? <ins>(1995)</ins> and Goosebumps <ins>(1996)</ins>. His breakthrough film role was as a Jewish neo-Nazi in The Believer <ins>(2001)</ins>, and he gained stardom for starring in the <ins>2004</ins> romantic drama The Notebook.</p>
```
Comment (eamzea) [inline index.html:16]: This code doesn't look correct, heading inside paragraphs? strong inside headings?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (eamzea) [inline index.html:17]: The readability of this code is poor

File: index.html
Comment (eamzea) [inline index.html:1]: I'm pretty sure this is not the correct place where the biography/file should be

File: 02/index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <p><h1><strong>My autobiography</strong></h1></p>
+<p>
+    <h2><strong>Name:</strong> <em>Abril Muñoz</em></h2>
+    <strong>Age:</strong> 29 years old
+    <strong>Date of birth:</strong> April <mark>8</mark> 1994
+    <strong>Nationality:</strong> Mexican
+    <strong>Description:</strong> 'I'm Abril, I'm from Mexico, I'm 29 years old, I'm psychologist and I work in a big company called <i>"Adecco"</i>, my currrent role is <i>Tech Recruiter.</i> 
+    I live with my friend, her name is <ins>Viri,</ins> and we live close to the downtown to the City.
+</p>
+
```
Comment (eamzea) [inline 02/index.html:18]: The same as the other :D

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by soyedgarzea — CHANGES_REQUESTED** [review]
- Why there are two html, body, head tags

File: 02/index.html [hunk 4]
```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta name="description" content="My autobiography and the story about my life"
+    <meta name="Description" content="My autobiography and the biography about Ryan Gosling">
```
Comment (eamzea) [inline 02/index.html:4]: Meta tags name should be lowercase

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**